src/plasma/plasma_store.cc

```diff
@@ -231,7 +232,8 @@ int create_object(Client *client_context,
     return PlasmaError_OutOfMemory;
   }
   /* Allocate space for the new object */
-  uint8_t *pointer = (uint8_t *) dlmalloc(data_size + metadata_size);
+  uint8_t *pointer =
+      (uint8_t *) dlmemalign(BLOCK_SIZE, data_size + metadata_size);
```

Comment:
we should document why we are doing this, specifically that 64-byte alignment is REQUIRED by `compute_object_hash_parallel`